pyramus/src/main/webapp/templates/students/editstudent.jsp

```diff
@@ -1045,6 +1049,103 @@
+                  id="coursesTabRelatedActionsHoverMenuContainer.${student.id}"
+                  class="tabRelatedActionsContainer"></div>
+            <input type="hidden" name="studentCards" value="${studentCards[student.id]}"/>
+            
+            <div class="genericFormSection">
+              <jsp:include page="/templates/generic/fragments/formtitle.jsp">
+                <jsp:param name="titleLocale" value="integrations.slice.title"/>
+              </jsp:include>
+            </div>
+            
+            <div class="genericFormSection">
+              <jsp:include page="/templates/generic/fragments/formtitle.jsp">
+                <jsp:param name="titleLocale" value="students.editStudent.firstNameTitle"/>
+              </jsp:include>            
+              ${person.latestStudent.firstName}
+            </div>
+            
+            <div class="genericFormSection">  
+              <jsp:include page="/templates/generic/fragments/formtitle.jsp">
+                <jsp:param name="titleLocale" value="students.editStudent.lastNameTitle"/>
+              </jsp:include>            
+              ${person.latestStudent.lastName}
+            </div>
+            
+            <div class="genericFormSection">   
+              <jsp:include page="/templates/generic/fragments/formtitle.jsp">
+                <jsp:param name="titleLocale" value="students.editStudent.birthdayTitle"/>
+                <jsp:param name="helpLocale" value="students.editStudent.birthdayHelp"/>
+              </jsp:include>  
+                <fmt:formatDate value="${person.birthday}" />
+            </div>
+            
+            <div class="genericFormSection">   
+              <c:forEach var="student" items="${students}">
+                <div class="genericViewInfoWapper ${secureInfoClass}"
+                  id="studentViewIntegrationInfoWrapper">
+              
+                  <div class="genericFormSection">           
+                    <jsp:include page="/templates/generic/fragments/formtitle.jsp">
+                      <jsp:param name="titleLocale" value="students.editStudent.studyProgrammeTitle"/>
+                      <jsp:param name="helpLocale" value="students.editStudent.studyProgrammeHelp"/>
+                    </jsp:include>            
+                    <div>${student.studyProgramme.name}<c:if test="${student.hasFinishedStudies}">*</c:if></div>
+                  </div>
+                  <div class="genericFormSection"> </div>
+                  <div class="genericFormSection">               
+                    <jsp:include page="/templates/generic/fragments/formtitle.jsp">
+                      <jsp:param name="titleLocale" value="students.editStudent.emailTableEmailsTitle"/>
+                    </jsp:include>
+                    
+                    <div id="emailTable.${student.id}"></div>
+                  </div>
+                  <c:set var="studentCard" value="${studentCards[student.id]}" />
```

Comment:
Does this really work? If the student doesn't have a card, wouldn't accessing this variable later on cause crashes?